code/components/citizen-server-impl/src/state/ServerGameState.cpp

```diff
@@ -4379,6 +4379,22 @@ struct CRemoveWeaponEvent
     MSGPACK_DEFINE_MAP(pedId, weaponType);
 };
 
+struct CRemoveAllWeaponsEvent
```

Comment:
Would be good if you add a typedoc documentation for this event. See "startProjectileEvent".